app/Console/Commands/RosterUpdate.php

```diff
@@ -129,6 +129,12 @@ public function handle() {
 
         $users = User::where('status', 2)->get();
         foreach ($users as $u) {
+            if ($u->hasRole('ins')) {
+                $u->detachRole('ins');
+            }
+            if ($u->hasRole('mtr')) {
+                $u->detachRole('mtr');
+            }
```

Comment:
Are these the only 2 roles that could possibly be assigned to someone that has this status? If not and we're just depending on that to be done manually, could we just detach all roles as a failsafe?

Reply:
@iccowan Great point - I've replaced this logic with `$u->removeRoles();` which will remove all roles from inactive users.